src/styles/blocks/product-card.scss

```
@@ -0,0 +1,82 @@
+.product-card {
+  width: 200px;
+  background-color: white;
+  border-radius: 5px;
+  border: 1px #f3f3f3 solid;
+  box-sizing: border-box;
+  padding: 33px 16px 16px;
```

```suggestion
  padding: 32px 16px 16px;
```
i suggest it should fix the tests

---

I changed the picture format from SVG to JPEG and hoped that the tests would pass, but nothing changed. This is where the 1 pixel offset comes from. In the chat mentor told me that with such a minimal difference with the tests, i can leave it as is and do the PR.